src/routes/Messages/MessageSection/MessageSection.test.js

```diff
@@ -55,105 +55,29 @@ const messages = [
 let wrapper, instance, socket
 
 beforeEach(() => {
-  socket = { on: jest.fn() }
-})
-
-it('renders as expected', () => {
```

Comment:
should we file a ticket to redo these tests?

Reply:
Meh. Honestly, I don't know that I'd bother in this case.